src/SymbolCollector.Android.Library/AutoUploader.cs

```diff
@@ -14,14 +15,14 @@ namespace SymbolCollector.Android.Library
     /// Automatically collects symbols with sentry.io's collector server.
     /// </summary>
     [Register("io/sentry/symbolcollector/AutoUploader")]
-    public class AutoUploader : Java.Lang.Object, Java.Lang.IRunnable
+    public class AutoUploader : Java.Lang.Object
     {
         /// <summary>
         /// Run symbol collection.
         /// </summary>
-        public void Run()
+        public void Run(Context context)
```

Comment:
This was done to be called from Java. The goal was to create an `aar` to be shared with customers so they can collect symbols for us. This project never went live so not even sure if we can call into this method from Java: Need to test this `Register` hook